### ISSUE TITLE
Deletions are not synced

### ISSUE THREAD

**Issue by danielo515** [wshallum/couchadaptor#15]
When you have two wikis that syncs to the same couchdb newly created tiddlers are correctly synced between two, but deletions does not.

I don't know how to understand this as a feature or as a problem. There is only one scenario where I'm totally sure this is a problem: in Draft management. Drafts are temporal by definition and once you have finished edditing you don't want that draft anymore, and of course you don't want a draft to be kept on several instances of TW.

To reproduce the Draft problem:
* Open a tiddlywiki that syncs to a couchdb in two different computers or two different tabs.
* Edit a tiddler or create a new tiddler and leave it in draft mode long enough to be synced
* Wait in the other instance of the wiki to the next "skinny tiddlers" fetch
* Draft appears on recent tab, absolutely empty unless you open it.
* Finish editing on the first wiki. Draft is deleted from DB
* Open the draft on the second instance. You have an empty draft, this can cause confusion because you also have the normal tiddler.
* Try to finish editing on the second instance with the empty draft and you will get an error because that document does not exist on the database anymore

I'm not sure how to address this problem. Avoid draft synchronization should not be an option, because I strongly thing that drafts should be synced.

**Comment by wshallum** [comment]
Yes, this currently does not work since the skinny tiddler list is a list of *existing* tiddlers. There seems to be no way in the current architecture for the server to notify the TW client that some tiddlers have been deleted on the server.

I personally don't have any intention of investigating new ways of doing this in TW. I would rather have other people investigate how to handle this in e.g. the Tiddlyweb adaptor and follow their lead. Getting info of deleted documents is theoretically possible using `_changes`: http://docs.couchdb.org/en/1.6.1/api/database/changes.html

This is of course most visible with draft tiddlers but it would be the same with non-draft tiddlers.

**Comment by wshallum** [comment]
[Danielo's thread on tiddlywikidev](https://groups.google.com/forum/?fromgroups=#!topic/tiddlywikidev/fKA8cllDhR4)
